webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -1144,38 +1144,32 @@ <h4>Collector</h4>
                     <th scope="row">SubmittedFromInfobar</th>
                     <td>{{ raw.SubmittedFromInfobar|default("--") }}</td>
                   </tr>
-                  <tr>
-                    <th scope="row">Payload</th>
-                    <td>{{ raw.payload }}</td>
-                  </tr>
-                  <tr>
-                    <th scope="row">Payload compressed</th>
-                    <td>{{ raw.payload_compressed }}</td>
-                  </tr>
-                  <tr>
-                    <th scope="row">Dump checksums</th>
-                    <td>
-                      {% if raw.dump_checksums %}
-                        <table class="data-table">
-                          {% for name, sha in raw.dump_checksums.items() %}
-                            <tr>
-                              <th class="w-2/12">{{ name }}:</th>
-                              <td>
-                                <code>{{ sha }}</code>
-                                {% if sha == "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855" %}
-                                  <span class="humanized">(empty minidump)</span>
-                                {% endif %}
-                              </td>
-                            </tr>
-                          {% endfor %}
-                        </table>
-                      {% endif %}
-                    </td>
-                  </tr>
-                  <tr>
-                    <th scope="row">Collector notes</th>
-                    <td><pre>{{ raw.collector_notes|join("\n") }}</pre></td>
-                  </tr>
+                  {% for key, val in raw.metadata | dictsort %}
```

Comment:
Redoing it so that it shows everything in `metadata` makes it easier to add new fields since we only have to add them in Antenna and don't have to do anything additionally here.